src/Microsoft.AspNet.Mvc.Core/ChallengeResult.cs

```diff
@@ -5,9 +5,29 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Microsoft.AspNet.Http.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.AspNet.Mvc
 {
+    internal static class ChallengeResultLoggerExtenstions
```

Comment:
This should be its own file, and don't put it in the `Microsoft.AspNet.Mvc` namespace, suggest `Microsoft.AspNet.Mvc.Logging`